### ISSUE TITLE
Invalid ACMode

### ISSUE THREAD

**Issue by RMCob** [sampsyo/wideq#18]
Got the following error:
$ python3 example.py ac-mon d27bdb00-7149-11d3-80b0-2c2bf923c76c 
Traceback (most recent call last):
  File "example.py", line 203, in <module>
    main()
  File "example.py", line 199, in main
    example(args.country, args.language, args.cmd, args.args)
  File "example.py", line 163, in example
    example_command(client, cmd, args)
  File "example.py", line 139, in example_command
    func(client, *args)
  File "example.py", line 93, in ac_mon
    'on' if state.is_on else 'off'
  File "/home/pi/Desktop/wideq-1.0.1/wideq.py", line 1059, in mode
    return ACMode(self.lookup_enum('OpMode'))
  File "/usr/lib/python3.5/enum.py", line 241, in __call__
    return cls.__new__(cls, value)
  File "/usr/lib/python3.5/enum.py", line 476, in __new__
    raise ValueError("%r is not a valid %s" % (value, cls.__name__))
ValueError: '@AC_MAIN_OPERATION_MODE_ENERGY_SAVER_W' is not a valid ACMode

**Comment by sampsyo** [comment]
Interesting! I based the list of modes:
https://github.com/sampsyo/wideq/blob/b176cd26eea01221d40d225c11d342d468b4b28a/wideq.py#L819-L827

on the modes that the API reports as being support by my model. `_ENERGY_SAVING_W` is definitely one of them. But it looks like yours supports something called `_ENERGY_SAVER_W`. Another testament to the beauty and consistency of this API. 🙄

I guess we could consider just adding another entry for this mode, and not trying to reconcile the two names?

**Comment by RMCob** [comment]
Works for me.

**Comment by jsmath** [comment]
I am having the same issue. I just ran example.py with the "mon" switch on my system and I it prints out "OpMode: @AC_MAIN_OPERATION_MODE_ENERGY_SAVER_W"
So it looks like I could benefit from this change as well.  I forked my own repo of this tried to change line 827 shown above to "ENERGY_SAVING = "@AC_MAIN_OPERATION_MODE_ENERGY_SAVER_W", but I don't think it is pulling in my new wideq repo even though I have tried updating the "REQUIREMENTS" in climate.py multiple times.  Also tried following the formatting instructions for the repo REQUIREMENTS here:
https://developers.home-assistant.io/docs/en/0.85/creating_component_deps_and_reqs.html

Any help would be appreciated!  Thanks for all of you great work on this component!
